service/src/main/java/uk/gov/hmcts/reform/fpl/handlers/ManageDocumentsUploadedEventHandler.java

```diff
@@ -357,9 +359,13 @@ private boolean isHearingDocument(DocumentType documentType) {
     @EventListener
     public void createWorkAllocationTask(ManageDocumentsUploadedEvent event) {
         CaseData caseData = event.getCaseData();
+
+        boolean shouldCheckCtscLevelChange = userService.isJudiciaryUser() || userService.isCafcassUser();
+
         if (isNotEmpty(event.getNewDocuments().get(COURT_CORRESPONDENCE))
             || isNotEmpty(event.getNewDocumentsLA().get(COURT_CORRESPONDENCE))
-            || isNotEmpty(event.getNewDocumentsCTSC().get(COURT_CORRESPONDENCE))) {
+            || (shouldCheckCtscLevelChange
```

Comment:
think a CTSC can upload a non-confidential document which might go into event.getNewDocuments() if I'm not mistaken, so might be missed by this condition

Reply:
Yes, right,. Updated the checking to skip adding WA task if it is HmctsAdmin user